src/modules/keyboardq/keyboard.qml

```diff
@@ -0,0 +1,204 @@
+import io.calamares.modules 1.0 as Modules
+import io.calamares.ui 1.0
+
+import QtQuick 2.10
+import QtQuick.Controls 2.10
+import QtQuick.Layouts 1.3
+import org.kde.kirigami 2.7 as Kirigami
+
+ResponsiveBase
```

Comment:
This is a QML component that Calamares itself should provide via its standard QML files, right?

Reply:
probably yes, it would make things a lot simpler to write the rest of qml views